app/Repositories/ReportbackRepository.php

```diff
@@ -134,4 +135,20 @@ public function getReportbackByUser($campaignId, $campaignRunId, $userId, $type
 
         return Reportback::where($parameters)->first();
     }
+
+    /**
+     * Updates an individual reportbackitem or many reportback items.
+     *
+     * @param array $data
+     *
+     * @return
+     */
+    public function updateReportbackItems($data)
+    {
+        foreach ($data as $reportbackItem) {
+            $rbItem = ReportbackItem::where(['id' => $reportbackItem['rogue_reportback_item_id']])->first();
```

Comment:
For example (from my comment above), YOu should check that `$reportbackItem['rogue_reportback_item_id']` exists and is not empty here as a safety measure.